src/data/markdown/docs/02 javascript api/11 k6-execution/1-Execution.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Execution
+excerpt: 'Get execution information'
+---
+
+k6 v0.34.0 introduces the capability for getting information around the execution process from the test's script.
+
+It can be used importing the `k6/execution` module.
+
+```
+import exec from 'k6/execution';
+
+var options = {
+	scenarios: {
+		'myscenario': { // this will be the returned name
+			executor: 'shared-iterations',
+			maxDuration: 30m
+		}
+	}
+}
+
+export default function() {
+	console.log(exec.scenario.name) // myscenario 
+}
+```
+
+The module has a default export that has the following keys `scenario`, `instance` and `vu` which in themselves expose information specific to the context in which it's being accessed.
+
+### Scenario
```

Comment:
If you don't look at the examples, it's not clear that the actual path to these properties is something like `k6/execution.scenario.name`... Not sure even if this change is enough. Any ideas @grafana/k6-devs?
```suggestion
### `scenario`
```

Reply:
I prefer leaving it as it was before, since this doesn't make that point clearer. We do mention the keys of the exported object in the paragraph before this, so with the examples I think the usage should be clear enough.